yaml.properties

```diff
@@ -7,6 +7,12 @@ publicVersions=1.4.0
 defaults.mavenGroupId=com.github.sbaudoi
```

Comment:
You need to move `1.4.0` from `publicVersions` to `archivedVersions`, and put `1.4.1` in `publicVersions`
(I can't make this comment on the appropriate lines because they're not part of the changeset :-/)